liquibase-couchbase/src/test/java/liquibase/ext/couchbase/operator/ClusterOperatorTest.java

```diff
@@ -216,7 +218,10 @@ void should_return_true_if_collection_index_exist() {
         when(collectionQueryIndexManager.getAllIndexes()).thenReturn(Collections.singletonList(queryIndex));
         when(queryIndex.name()).thenReturn(TEST_INDEX);
 
-        boolean result = clusterOperator.getCollectionOperator(collection).collectionIndexExists(TEST_INDEX);
+        boolean result = clusterOperator
```

Comment:
Looks like we could extract `clusterOperator
                .getBucketOperator(TEST_BUCKET)
                .getCollectionOperator(TEST_COLLECTION, TEST_SCOPE)` to class constant, could we?